dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -132,6 +132,19 @@
       https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1296
     </Reason>
   </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="reactor.blockhound" />
+    </Target>
+    <Source>
+      <Class name="io.netty.util.internal.Hidden" />
+    </Source>
+    <Reason>
+      Netty users can optionally add BlockHound dependency to detect blocking calls in their usage
+      of netty. The io.netty.util.internal.Hidden class implements the integration and this class
```

Comment:
So if they don't add this class, then there are linkage errors?

Reply:
Right. If Netty users don't add the optional BlockHound dependency, then Linkage Checker detects linkage errors in io.netty.util.internal.Hidden class has linkage errors. This won't manifest as runtime errors.

Reply:
I know we've seen this before. Do you recall why we didn't do this then? 

We should probably VC and screenshare to figure out how (or whether) this code is guarded.

Reply:
Netty itself doesn't seem to call this Hidden class outside of tests. It's not obviou to me who does. It's only accessed via a service loader.

Reply:
Perhaps Blockhound itself is the thing that loads this? Netty just makes it available to be loaded?

Reply:
https://github.com/reactor/BlockHound/blob/master/docs/custom_integrations.md

Reply:
I think this is OK, but the reason needs to be clearer and less generic. E.g. "This class is only loaded by Blockhound itself, in which case the missing classes are on the classpath. If Blockhound is not on the classpath, nothing loads this class."

Reply:
Thanks. Updated the explanation with the URL.